kolibri/utils/options.py

```diff
@@ -22,14 +22,19 @@ def calculate_thread_pool():
     Servers with more memory can deal with more threads.
     Calculations are done for servers with more than 2 Gb of RAM
     """
-    pool = 10
+    MIN_POOL = 50
+    MAX_POOL = 100
+    pool = MIN_POOL
     if psutil:
+        min_mem = 2
+        max_mem = 4
         total_memory = psutil.virtual_memory().total / pow(2, 30)  # in Gb
-        if total_memory > 2:
-            pool = int(45 * total_memory - 80)  # 10 for 2 Gb,100 for 4 Gb
-            pool = 200 if pool > 200 else pool
+        if min_mem < total_memory < max_mem:
+            pool = int(MAX_POOL * float(total_memory) / max_mem)
```

Comment:
This I think works because `MIN_POOL / MAX_POOL` currently equals `min_mem / max_mem` but I believe it may not work as expected otherwise. perhaps something like:

```python
MIN_POOL + (MAX_POOL - MIN_POOL) * float(total_memory) / max_mem
```

Reply:
Yes, can update.